addon/mirage/factory.js

```diff
@@ -1,3 +1,21 @@
+import Ember from 'ember';
+
+function generateValue(type, key) {
+  if (type === 'string') {
+    return (i) => `${key} #${i}`;
+  } else if (type === 'number') {
```

Comment:
Expected an identifier and instead saw '}'.<br>Missing semicolon.<br>Expected an identifier and instead saw 'else'.<br>Expected an assignment or function call and instead saw an expression.